### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by ccrowles** [Wikidepia/InstaFix#130]
Not working anymore with posts, at all.
Not even `d.dd` works.

![Screenshot_20241018-102832](https://github.com/user-attachments/assets/ce4b59c9-6580-480b-ab12-67fbc0b9f8fe)

**Comment by Wikidepia** [comment]
It should work now, can you try it again?

**Comment by ccrowles** [comment]
yep, now it works, thanks!

**Comment by ayurai** [comment]
> It should work now, can you try it again?

![image](https://github.com/user-attachments/assets/9f7d3725-2e54-41c9-b4ae-20b0ef125479)
why most of links now is going this way?

**Comment by Haxim** [comment]
back to not working again

![image](https://github.com/user-attachments/assets/25bb2f86-d396-4b0b-b716-53e7860ddea6)

**Comment by TheMasterOfMadness** [comment]
Agreed, it's not working anymore.

**Comment by patros36** [comment]
same, not working for me as well

**Comment by TheAGames10** [comment]
It is now November 14th, and the embed links for InstaFix are not working anymore, it's also doing what is displayed in this issue.

**Comment by ayurai** [comment]
yeah, doesn't look like it's working anymore, I've even tried with links that work before , but nothing (on telegram at least)

**Comment by Elapsed** [comment]
So I can't speak to why the dev hasn't been around in the last month. But I can, I believe partly explain what has happened (not that it's terribly helpful to an end-user, trust me, I'm an end-user and knowing this hasn't helped much)

A little over a month ago now, Meta changed the authentication stuff on the Instagram side of things. It made it significantly harder to use bot accounts to pull the post for these embeds on Discord and Telegram. Sometimes it requires an actual human to interact and it's a bit weird with location information.

So I know this, because [FixThreads ](https://github.com/milanmdev/fixthreads) was having the same issues. I was reporting and updating that dev via their Discord server about it. And they've made a bit of a workaround. But I don't know what that workaround is, and if it would work for InstaFix when the dev returns to the project. InstaFix undoubtedly has a LOT more users and maybe the same fix for the Threads fix wouldn't be as comprehensive as needed

Anyhow I hope Wikidepia returns and gives us an update. For now I just need to let my server know Instagram links may have videos or more pictures and need clicking

**Comment by patros36** [comment]
For now, you can use "kk" instead of "dd" to embed instagram posts...used it yesterday and it works....don't know where it is from, couldn't find a github page or anything, but it works.

**Comment by Fatez94** [comment]
In Italy doesn't work on telegram !!!! Please fix it ♥️
